acbuild/acbuild.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2015 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	cliName        = "acbuild"
+	cliDescription = "acbuild, the application container build system"
+)
+
+var (
+	tmpaci       = ".acbuild.tmp"
+	targetpath   = ".target"
+	scratchpath  = ".scratch"
+	depstorepath = ".depstore"
+	workpath     = ".work"
+
```

Comment:
These are created in the cwd right? So you have context mode for free and can have multiple build just executing acbuild in different dirs right? I like it.

* What about using a parent dir (something like `.acbuild`) to contain them?
* What about an option to specify the path of the build (if not specified will default to the cwd)